lib/bitreserve/base.rb

```diff
@@ -0,0 +1,15 @@
+module Bitreserve
+  class Base
+    def initialize(attributes = {})
+      instantiate_variables(attributes)
+    end
+
+    private
+
+    def instantiate_variables(attributes)
+      attributes.each_pair do |key, value|
+        instance_variable_set("@#{key}", value)
+      end
+    end
+  end
+end
```

Comment:
Shouldn't this be in a separate directory? Base and Ticker seem off-place here. At first I was having trouble understanding what was this a base for